### PR TITLE
trim v on version check

### LINKED ISSUE #3034: Update notice showing for v2.1.0 while already on v2.1.0
After switching to the v2.1.0 docker image for woodpecker, the admin interface says to update to the current version.

image

This is from the docker image: `woodpeckerci/woodpecker-server:v2.1.0`

### PR DESCRIPTION
**Pull request by 6543**
close #3034

### CODE REVIEW

**Comment by 6543** [comment]
a simple hotfix to release a v2.1.1

File: web/src/compositions/useVersion.ts [hunk 1]
```diff
@@ -72,9 +72,9 @@ export function useVersion() {
       if (usesNext) {
         latest = versionInfo.next;
       } else if (current.includes('rc')) {
-        latest = versionInfo.rc;
```
Comment (anbraten) [inline web/src/compositions/useVersion.ts:75]: @6543 Or should we directly do this in the ci config file updating the version.json file.

Comment (6543) [reply web/src/compositions/useVersion.ts:75]: I want to release a v2.1.1 anyway ... :)

and later i suggest to use a version parse lib

Comment (6543) [reply web/src/compositions/useVersion.ts:75]: #3041

**Review by xoxys — APPROVED** [review]

**Pull request merged**